### DISCUSSION TITLE
Default LFS flag true

### DISCUSSION THREAD

**Discussion by JamesIves** [JamesIves/github-pages-deploy-action#474] [Ideas]
I'm wondering if the `LFS` flag should even be an option, or if the LFS migration command should _always_ run or not. I'd like to simplify the API slightly with 4.0.0.

**Comment by JamesIves** [comment]
PR to make this happen is here: https://github.com/JamesIves/github-pages-deploy-action/pull/503

**Reply by Pike** [reply]
The `lfs` part is another thing I'm trying to grok right now, sorry. How does `lfs` leak into the deployed branch? Is that through `rsync` copying over a `.gitattributes` that tries to enable that?

If so, should the attempt to export run on the temporary deploy worktree instead of the upstream one?

**Reply by JamesIves** [reply]
Related discussion: https://github.com/JamesIves/github-pages-deploy-action/issues/373
